catalog/src/main/kotlin/com/adevinta/spark/catalog/model/Components.kt

```diff
@@ -333,24 +348,24 @@ private val Tokens = Component(
 
 /** Components for the catalog, ordered alphabetically by name. */
 public val Components: List<Component> = listOf(
-    Tokens,
```

Comment:
Tokens are at the top since they are a core atoms from the DS, we can let them at this place. However the components should be ordered alphabetically